src/Models/ElementalArea.php

```diff
@@ -25,7 +25,7 @@
 class ElementalArea extends DataObject
 {
     private static $db = [
-        'OwnerClassName' => 'Varchar(255)',
+        'OwnerClassName' => 'DBClassName("SilverStripe\\\\ORM\\\\DataObject")',
```

Comment:
Should it be `DBClassName("SilverStripe\\\\ORM\\\\FieldType\\\\DBVarchar")` to keep it effectively a Varchar? Don't understand the change to DataObject.

Reply:
That's the base class that DBClassName should use. The Value of 'OwnerClassName' is a subclass of dataobject.

Reply:
@michalkleiner I've just skim read this, but the change from varchar to DBClassName is to make it a proper polymorphic relationship, which means that dev/build in framework can perform modifications on it like it does for other polymorphic relations.

Reply:
We should build a proper polymorphic belongs_to in core right @robbieaverill ? :P

Reply:
Yeah! I guess it this were replaced with Owner -> DataObject then it'd be even more real